packages/ProgressAccordion/src/ProgressAccordion.js

```diff
@@ -22,25 +22,24 @@ const propTypes = {
 
   /** List items (must be of type <ProgressAccordion.Item>. */
   children: PropTypes.node,
-
-  /** When kind is "navigation", all the children are expanded and cannot be collapsed */
-  kind: PropTypes.oneOf(["", kinds.NAVIGATION]),
 };
 
 const defaultProps = {
   a11yText: null,
   activeIndex: 0,
   activeStatus: null,
   children: null,
-  kind: "",
 };
 
 function filterChildren(children) {
-  return React.Children.toArray(children).filter(child => child.type && child.type.displayName === Item.displayName);
+  return React.Children.toArray(children).filter(
+    child =>
+      child.type && (child.type.displayName === Item.displayName || child.type.displayName === NavItem.displayName)
```

Comment:
I think you could just reexport the `Item` as NavItem and you wouldn't need to change anything and everything would keep working.

ProgressAccordion.Item = Item;
ProgressAccordion.NavItem = Item;

what you are trying to do is improve the consumer side, no need to modify the author part.

Reply:
As discussed they behave slightly differently, and putting in two separate components is cleaner than having one that has `if (kind === 'navigation')` throughout.

Reply:
`child => [Item.displayName, NavItem.displayName].includes(child.type?.displayName)`

shorter?